lib/rubocop/cop/style/mutable_constant.rb

```diff
@@ -12,6 +12,11 @@ module Style
       #
       #   # good
       #   CONST = [1, 2, 3].freeze
+      #
+      #   # good
+      #   CONST = <~TESTING.freeze
```

Comment:
It has a syntax error, could you replace `<~` with `<<~` and re-run `rake generate_cops_documentation`?